Modules/Filtering/Convolution/include/itkConvolutionImageFilterBase.hxx

```diff
@@ -112,7 +112,23 @@ ConvolutionImageFilterBase<TInputImage, TKernelImage, TOutputImage>::PrintSelf(s
   Superclass::PrintSelf(os, indent);
 
   os << indent << "Normalize: " << m_Normalize << std::endl;
-  os << indent << "BoundaryCondition: " << m_BoundaryCondition->GetNameOfClass() << std::endl;
+  os << indent << "DefaultBoundaryCondition: " << m_DefaultBoundaryCondition << std::endl;
+  if (m_DefaultBoundaryCondition)
```

Comment:
@thewtex Cannot remember if there was a shorter way to print objects that do not inherit from `LightObject`. Thanks.

Reply:
@jhlegarreta nothing that I recall, either. This LGTM

Reply:
:+1: